content/en/docs/ops/configuration/traffic-management/dns/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Understanding DNS
+linktitle: DNS
+description: How DNS interacts with Istio.
+weight: 30
```

Comment:
It looks like we have several pages using the same weight. I'd at least pick one here so that the order is deterministic.